mypy/plugins/common.py

```diff
@@ -154,7 +154,7 @@ def find_shallow_matching_overload_item(overload: Overloaded, call: CallExpr) ->
                 ):
                     ok = False
                     break
-                elif isinstance(arg_type, LiteralType) and type(arg_type.value) is bool:
+                elif isinstance(arg_type, LiteralType) and isinstance(arg_type.value, bool):
```

Comment:
What caused this change?

Reply:
```
mypy\plugins\common.py:157:60: E721 Do not compare types, use `isinstance()`
Found 1 error.
```

Reply:
I don't personally find this lint rule particularly valuable so would also be fine with just ignoring it in our pyproject.toml config. But also, in this specific instance, the semantics are exactly the same since `bool` cannot be subclassed. So it felt easier just to genuflect to the linter here.

Reply:
Ah ok, as long as it was _unintentional_ exact check.

Reply:
What I'm saying is that for `bool` specifically, the "exact" and the "inexact" check come to exactly the same thing. Since `bool` cannot be subclassed, using `isinstance()` here is just as good a way of testing whether the type of `arg_type.value` is "exactly the `bool` class"